### DISCUSSION TITLE
Can't modify data from other backend users

### DISCUSSION THREAD

**Discussion by kopieMK** [pimcore/pimcore#14147] [Q&A / Support] [answered]
Hello,
our Pimcore version is currently 10.4.0. I have the problem that I can't modify data from other backend users. 
I use an administrator account. I can only change the data for the user I am logged in with.
Ex:
I change the role or name of a user and save. I get a message that the save was successful. If I close the tab of the user and open it again, these changes are not taken over.
But I can change the data of my logged in admin without any problems.

I currently don't know at all where to start looking or what the problem could be. 
Does anyone have any ideas what I could try out to find the problem? Or does anyone even know of a solution to this?

Thanks for your support!

Magnus
![0002](https://user-images.githubusercontent.com/50910488/214821687-21aa5376-c4d7-42d4-a53d-c550c001f319.gif)

**Comment by kingjia90** [comment, marked as answer]
Please see https://github.com/pimcore/pimcore/issues/12138 , by updating to  10.4.1 should fix it

**Comment by sngm** [comment]
OMG! Yes this works. Thank you and have a nice day!

**Reply by kopieMK** [reply]
Works! TY (Sorry above was my other Account)
